code/modules/hydroponics/plants_weed.dm

```diff
@@ -158,7 +158,7 @@ ABSTRACT_TYPE(/datum/plant/weed)
 					radstrength = 50
 					radrange = 3
 			for (var/mob/living/carbon/M in range(radrange,POT))
-				M.changeStatus("radiation", (radstrength) SECONDS, 3)
+				M.take_radiation_dose(radstrength/30 SIEVERTS)
```

Comment:
oh jeez these were pretty crazy even before, weren't they.
A single dose even at max potency wont' be lethal, but any more than that and people are going to have a bad time
Might want to cheat a bit and put a cooldown on the victim so they can't be slammed by a dozen stacked plants perhaps?